apps/core/components/product-card-carousel/index.tsx

```diff
@@ -45,7 +45,7 @@ export const ProductCardCarousel = ({
   }, []);
 
   return (
-    <Carousel aria-labelledby="title" className="mb-14">
+    <Carousel aria-labelledby="title" className="mb-14" opts={{ loop: true }}>
```

Comment:
Are we changing this back to `true`?

Reply:
I had the same thought, but I was thinking loop was disabled for the slideshow only because of CLS. Was the product carousel also causing CLS @jorgemoya?

Reply:
It was but I don't see any artifacts now when running locally or on the build.

Reply:
Late here, but chiming in to confirm it seems to have vanished 🤷 The CLS *itself* has shifted!